library/src/main/java/com/mux/player/internal/cache/CacheDatastore.kt

```diff
@@ -283,9 +381,12 @@ internal class CacheDatastore(val context: Context) : Closeable {
 
       val helper = DbHelper(context, indexDbDir())
       closeIfInterrupted(helper)
-      val db = helper.writableDatabase
-      db.close()
-      // todo- eviction pass with that db
+
+      // Do some db maintenance when we start up, in case shutdown wasn't clean
+      helper.writableDatabase.use { db ->
```

Comment:
Curious if this was added solely as a precaution or did you spot something during testing when this wasn't done?

Reply:
Just a precaution